src/ethereum/LANDTerraformSale.js

```diff
@@ -0,0 +1,39 @@
+import * as env from "../env";
+// import eth from "./eth";
+import { Log } from "../log";
+
+import Contract from "./Contract";
+import { abi } from "../contracts/LANDTerraformSale.json";
+
+const log = new Log("[LANDTerraformSale]");
+let instance = null;
+
+/** LANDTerraformSale contract class */
+class LANDTerraformSale extends Contract {
+  static getInstance() {
+    if (!instance) {
+      instance = new LANDTerraformSale(
+        "LANDTerraformSale",
+        env.get("TERRAFORM_CONTRACT_ADDRESS", ""),
```

Comment:
Either pick an address (testnet or prodution - there are pros and cons for both options) or use a function that throws to prevent an empty value here (env.get allows a fallback to be a function, see https://github.com/decentraland/commons/blob/master/src/env.js#L70)

Reply:
I think it's better to throw than using a default value in this case,